src/zenml/cli/utils.py

```diff
@@ -187,8 +185,8 @@ def print_stack_configuration(
     )
     rich_table.add_column("COMPONENT_TYPE")
     rich_table.add_column("COMPONENT_NAME")
-    items = ([typ.value, name] for typ, name in config.items())
-    for item in items:
+    items = {typ.value: name for typ, name in config.items()}
```

Comment:
```suggestion
    items = {type.value: name for type, name in config.items()}
```

Reply:
Ah, I really hate naming variables that shadow builtins in python..... would you prefer `type_`?